docs/docs/cli.md

```diff
@@ -1,91 +1,204 @@
-                                 less accurate.
-  --weight_download_region TEXT  Defaults to 'us', can also be 'eu' or 'asia'.
-                                 Region for server to download weights.
-  --verbose                      Displays additional logging information
-                                 during processing.
-  --help                         Show this message and exit.
+  --data-dir PATH                 Path to folder containing videos.
+  --file-list PATH                Path to csv containing `filepath` column
+                                  with videos.
+
+  --model [time_distributed|slowfast]
+                                  Model to use for inference.  [default:
+                                  time_distributed]
+
+  --checkpoint PATH               Model checkpoint path to use for
+                                  inference.     If using, model name must
+                                  also be specified.
+
+  --gpus INTEGER                  Number of GPUs to use for inference. If
+                                  not specifiied, will use all GPUs found
+                                  on machine.
+
+  --batch-size INTEGER            Batch size to use for training.
+  --save / --no-save              Whether to save out predictions to csv
+                                  file.
+
+  --dry-run / --no-dry-run        Runs one batch of inference to check for
+                                  bugs.
+
+  --config PATH                   Specify options using yaml configuration
+                                  file instead of through command line
+                                  options.
+
+  --proba-threshold FLOAT         Probability threshold for classification
+                                  between 0 and 1. If specified binary
+                                  predictions are returned with 1 being
+                                  greater than the threshold, 0 being less
+                                  than or equal to. If not specified,
+                                  probabilities between 0 and 1 are
+                                  returned.
+
+  --output-class-names / --no-output-class-names
+                                  If True, we just return a video and the
+                                  name of the most likely class. If False,
+                                  we return a probability or indicator
+                                  (depending on --proba_threshold) for
+                                  every possible class.
+
+  -y, --yes                       Skip confirmation of configuration and
+                                  proceed right to prediction.  [default:
+                                  False]
+
+  --help                          Show this message and exit.
 ```
 
-Let's go through these one by one.
+All of the above besides `file-list`, `model`, `yes`, and `checkpoint` can also be specified in a yaml file. Let's go through the flags one by one.
+
```

Comment:
Is this (`file-list`, `model`, `yes`, and `checkpoint`) an accurate list of the param that can't be specified in the yaml?